examples/platform-app/package.json

```diff
@@ -8,26 +8,17 @@
         "build": "d2-app-scripts build",
         "start": "d2-app-scripts start",
         "test": "d2-app-scripts test",
-        "lint": "yarn d2-style check",
+        "lint": "d2-style check",
         "lint:staged": "yarn lint --staged",
-        "format": "yarn d2-style apply",
+        "format": "d2-style apply",
         "format:staged": "yarn format --staged",
         "cy:start": "BROWSER=none yarn start",
-        "cy:open": "d2-utils-cypress open --appStart 'yarn cy:start'",
-        "cy:run": "d2-utils-cypress run --appStart 'yarn cy:start'",
-        "cy:capture": "d2-utils-cypress run --capture --appStart 'yarn cy:start' --serverMinorVersion 37",
-        "cy:stub": "d2-utils-cypress run --stub --appStart 'yarn cy:start' --serverMinorVersion 37",
+        "cy:open": "concurrently 'yarn:cy:start' 'd2-utils-cypress open' --kill-others",
+        "cy:run": "concurrently 'yarn:cy:start' 'd2-utils-cypress run' --kill-others",
+        "cy:capture": "concurrently 'yarn:cy:start' 'd2-utils-cypress run --capture --serverMinorVersion 37' --kill-others",
+        "cy:stub": "concurrently 'yarn:cy:start' 'd2-utils-cypress run --stub --serverMinorVersion 37' --kill-others",
```

Comment:
I think `concurrently` should be a devDependency of this example app and not be specified in the root package.json as that decouples the dependency from the workspace that uses it (unlike d2-style for example, which operates on the root level)

Reply:
Will manage this in the follow-up PR. :+1: